templates/build-ports.p2

```diff
@@ -20,6 +22,10 @@ _build_port()
 
 for jail in ${JAIL_NAMES}
 do
+  if [ "$1" == "-y" ]; then
+      _build_port $jail
```

Comment:
Yes. This way one can run the options step manually at some time and they will be picked up when the build is executed next. The idea is that the builds can run unattended.